services/libs/comp-sys/cargo-component/tests/trait_method_test/Components.toml

```diff
@@ -1,19 +1,19 @@
 [components]
-foo = { name = "foo" }
-bar = { name = "bar" }
+foo2 = { name = "foo2" }
+bar2 = { name = "bar2" }
```

Comment:
When some project depends on `aster-frame` from GitHub url, `cargo` will warn that it founds multiple packages with the same name. Since these crates are only for test input, I simply rename them.